resources/service.php

```diff
@@ -423,6 +423,18 @@
                 ]
             ]
         ],
+        'getPriceSuggestions' => [
```

Comment:
The topic of this pull request is the extension of `search` with the parameters `page` and `per_page`. The enhancement `getPriceSuggestions` should be send as a separate pul request.

Reply:
getPriceSuggestions was added automatically to the pull request when I updated my version on github

You guys can accept it or not, ill just keep using my version